src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/PledgeOrchestrator.cs

```diff
@@ -28,6 +39,9 @@ public IndexViewModel GetIndexViewModel(string encodedAccountId)
         public async Task<CreateViewModel> GetCreateViewModel(CreateRequest request)
         {
             var cacheItem = await RetrievePledgeCacheItem(request.CacheKey);
+            var levels = await _tagService.GetLevels();
+            var sectors = await _tagService.GetSectors();
+            var jobRoles = await _tagService.GetJobRoles();
```

Comment:
As there are no interdependencies these awaits could be asych rather than await'ing each individually, such as:
```suggestion
            var levelsTask = _tagService.GetLevels();
            var sectorsTask = _tagService.GetSectors();
            var jobRolesTask = _tagService.GetJobRoles();
			await Task.WhenAll(levelsTask, sectorsTask, jobRolesTask);
```

Reply:
Yes we should